src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4351,6 +4365,9 @@ public override bool PreVisit(CallNode node)
                 // Typecheck the invocation and infer the return type.
                 fArgsValid &= maybeFunc.HandleCheckInvocation(_txb, args, argTypes, _txb.ErrorContainer, out var returnType, out var nodeToCoercedTypeMap);
```

Comment:
What happens to the errors generated on this line?

Reply:
You are correct, there should have been temp error container here.